### PR TITLE
[CI:DOCS] Fix up example description of podman-logout.1.md.in

### PR DESCRIPTION
**Pull request by rhatdan**
#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

**Review by flouthoc — APPROVED** [review]
LGTM

File: docs/source/markdown/podman-logout.1.md.in [hunk 1]
```diff
@@ -37,17 +37,17 @@ Print usage statement
 
 ## EXAMPLES
 
-Remove login credentials for the docker.io registry from the authorization file
+Remove login credentials for the docker.io registry from the authorization file:
```
Comment (vrothberg) [inline docs/source/markdown/podman-logout.1.md.in:40]:
```suggestion
Remove login credentials for the docker.io registry from the authentication file:
```

Comment (vrothberg) [reply docs/source/markdown/podman-logout.1.md.in:40]: Technically we could also be using an auth helper.

File: docs/source/markdown/podman-login.1.md.in [hunk 2]
```diff
@@ -71,7 +71,7 @@ print detailed information about credential store
 
 Add login credentials for specified registry to the default authorization file.
```
Comment (vrothberg) [inline docs/source/markdown/podman-login.1.md.in:72]: Sorry, there seem to be more cases of s/authorization/authentiation/ in these man pages.

**Review by vrothberg — APPROVED** [review]
LGTM, thank you

**Pull request merged**